style.css

```diff
@@ -1,5 +1,26 @@
+/* Section */
 
-
+.container {
+    width: 80%;
```

Comment:
if you can add here `margin: auto and height: 525` our website will look great:)

Reply:
I added this changes